.gitignore

```diff
@@ -110,3 +111,4 @@ COLLIER.pc
 Himalaya.pc
 LoopTools.pc
 TSIL.pc
+GM2Calc.pc
```

Comment:
Yes, we should ignore it.

Reply:
Sorry, a double negative. Should this line be added? Because this PR has nothing to do with GM2Calc. So I was not sure why is it changing anything related to it.

Reply:
Ah, you're right, this has nothing to do with this PR. @uukhas if you have the time, please revert this change. It should be added in a separate PR.

Reply:
I've removed it.